dev/tools/controllerbuilder/generate.sh

```diff
@@ -25,8 +25,10 @@ cd ${REPO_ROOT}/dev/tools/controllerbuilder
 
 
 # DiscoveryEngine
+# TODO: We use service v1alpha instead of v1 because v1 has key feature missing. 
+
 go run . generate-types \
-    --service google.cloud.discoveryengine.v1 \
+    --service google.cloud.discoveryengine.v1alpha \
     --api-version discoveryengine.cnrm.cloud.google.com/v1alpha1 \
     --resource DiscoveryEngineDataStore:DataStore \
     --resource DiscoveryEngineDataStoreTargetSite:TargetSite \
```

Comment:
I think in one of my PRs I added the ability to make the proto type fully qualified (i.e. `--resource DiscoveryEngineDataStore:google.cloud.discoveryengine.v1alpha.DataStore`), precisely to enable mixing versions.